### PR TITLE
Added Kickstart UEFI custom ptable

### PR DESCRIPTION
**Pull request by lzap**
Tested it works fine.

### CODE REVIEW

File: partition_tables_templates/kickstart_uefi_custom.erb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+part /boot --fstype="ext4" --ondisk=sda --size=1024
+part pv_sda --fstype="lvmpv" --ondisk=sda --size=1024 --grow
+volgroup vg_sda --pesize=4096 pv_sda
+logvol swap --fstype="swap" --size=4096 --name=lv_swap --vgname=vg_sda
+logvol / --fstype="ext4" --percent=40 --name=lv_root --vgname=vg_sda
+#logvol / --fstype="ext4" --grow --name=lv_root --vgname=vg_sda
+
+<% else -%>
+  
+autopart <%= host_param('lv_root_percent') %>
```
Comment (ekohl) [inline partition_tables_templates/kickstart_uefi_custom.erb:33]: https://docs.fedoraproject.org/en-US/fedora/rawhide/install-guide/appendixes/Kickstart_Syntax_Reference/#sect-kickstart-commands-autopart

This doesn't look like it accepts a percentage. Am I missing something?

File: partition_tables_templates/kickstart_uefi_custom.erb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+clearpart --all --initlabel
+<% if @host.pxe_loader.include?("UEFI") -%>
+  
+part /boot/efi --fstype="efi" --ondisk=sda --size=200 --fsoptions="umask=0077,shortname=winnt"
+part /boot --fstype="ext4" --ondisk=sda --size=1024
+part pv_sda --fstype="lvmpv" --ondisk=sda --size=1024 --grow
+volgroup vg_sda --pesize=4096 pv_sda
+logvol swap --fstype="swap" --size=4096 --name=lv_swap --vgname=vg_sda
+logvol / --fstype="ext4" --percent=40 --name=lv_root --vgname=vg_sda
+#logvol / --fstype="ext4" --grow --name=lv_root --vgname=vg_sda
```
Comment (ekohl) [inline partition_tables_templates/kickstart_uefi_custom.erb:20]: Is the commented line here intentional?

File: partition_tables_templates/kickstart_uefi_custom.erb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+oses:
+- CentOS
+- Fedora
+- RedHat
+%>
+zerombr
+clearpart --all --initlabel
+<% if @host.pxe_loader.include?("UEFI") -%>
+  
+part /boot/efi --fstype="efi" --ondisk=sda --size=200 --fsoptions="umask=0077,shortname=winnt"
```
Comment (ekohl) [inline partition_tables_templates/kickstart_uefi_custom.erb:14]: Should sda be a parameter but with a default?

Comment (lzap) [reply partition_tables_templates/kickstart_uefi_custom.erb:14]: Definitely, I wanted to do this not sure why I pushed this WIP version.

**Comment by lzap** [comment]
Weird, this is some leftover, a WIP version. This was merged already as 2c00c5dd4bcd2eb9edb50016b2d62aba3483221b sorry about that.

**Pull request closed without merging**